src/CoreWCF.Primitives/src/CoreWCF/Telemetry/WcfInstrumentationConstants.cs

```diff
@@ -0,0 +1,19 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace CoreWCF.Telemetry;
+
+internal static class WcfInstrumentationConstants
+{
+    public const string RpcSystemTag = "rpc.system";
+    public const string RpcServiceTag = "rpc.service";
+    public const string RpcMethodTag = "rpc.method";
+    public const string NetHostNameTag = "net.host.name";
```

Comment:
Are these from the OTel Semantic conventions? I see `network.peer.address` and `network.peer.port` as the way RPC & Http define these.

https://github.com/open-telemetry/semantic-conventions/blob/main/docs/rpc/rpc-spans.md

If there are additional WCF specific ones needed, then we should start a conversation with @lmolkova.

Reply:
I just reused those from the Otel community packages. If you want them to be changed let me know

Reply:
Please match names to the semantic conventions - this is the common standard from OTel for naming span properties, metrics etc. I think most are correct, its the `net.*` that needs to change.

Reply:
done